src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -76,15 +76,9 @@ public function getConfigTreeBuilder()
                 ->fixXmlConfig('path')
                 ->children()
                     ->arrayNode('paths')
-                        ->example(['%kernel.project_dir%/src/Enum' => 'App\Enum'])
-                        ->useAttributeAsKey('path')
-                        ->arrayPrototype()
-                            ->beforeNormalization()
-                                ->ifString()->then(static function (string $v): array { return ['namespace' => $v]; })
-                            ->end()
-                            ->children()
-                                ->scalarNode('namespace')->isRequired()->cannotBeEmpty()->end()
-                            ->end()
+                        ->example(['App\Enum' => '%kernel.project_dir%/src/Enum'])
+                        ->useAttributeAsKey('namespace')
+                        ->scalarPrototype()
```

Comment:
Won't it be better to have this as array, just like doctrine types where you can have two properties specified (name and type). This would be a safeguard for future updates to add more properties to a mapping (eg. have another `domain` for a namespace)

Reply:
That's wouldn't be much an issue actually using normalization for BC compat if we want to support such cases later.

Reply:
Then I'm ok with this. Thank you